hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -75,10 +75,14 @@ execute_robot_test(){
   CONTAINER="$1"
   TEST="$2"
   TEST_NAME=$(basename "$TEST")
-  TEST_NAME=${TEST_NAME%.*}
+  TEST_NAME="$(basename $COMPOSE_DIR)-${TEST_NAME%.*}"
```

Comment:
shellcheck:25: note: Double quote to prevent globbing and word splitting. [SC2086]